src/Catty/DataModel/Bricks/Sound/PlaySoundAndWaitBrick.m

```diff
@@ -31,23 +31,6 @@ - (kBrickCategoryType)category
     return kSoundBrick;
 }
 
-#pragma mark - Copy
-- (id)mutableCopyWithContext:(CBMutableCopyContext*)context
```

Comment:
Similar to the "Look Bricks", the same applies to the "Sound Bricks". Please refer to my previous comment about the _SetBackgroundBrick_.